orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationTemplate.java

```diff
@@ -58,6 +68,19 @@ public class OrchestrationTemplate extends TemplateConfig {
   @Nullable
   List<ChatCompletionTool> tools;
 
+  /**
+   * Create a new template with the given messages.
+   *
+   * @param messages The messages to use in the template.
+   * @return The updated template.
+   */
+  @Nonnull
+  public OrchestrationTemplate withMessages(@Nonnull final Message... messages) {
```

Comment:
(Comment)

Personally I like vararg on convenience API.
Unfortunately we are using it inconsistently with other accessors.
It looks like we are lacking a coherent API style guide.